domain/pricing.go

```diff
@@ -19,6 +19,7 @@ const (
 	// CoinGeckoPricingSourceType defines the pricing source
 	// that calls CoinGecko API.
 	CoinGeckoPricingSourceType
+	NoneSourceType
```

Comment:
Should we instead make `NoneSourceType` as the first element or explicitly set it to -1?

The problem with the current approach is that if we keep adding more types in the future and placing them before `NoneSourceType` in the enum, the value of `NoneSourceType` will keep increasing with every addition, potentially leading to breaking changes.

Reply:
I'll explicitly set `NoneSourceType` to `-1` and place it at the end, so that `ChainPricingSourceType` always stays at `0`, `CoinGeckoPricingSourceType` stays at `1`, and if we have more pricing sources in future, they will be 2, 3, 4...